pom.xml

```diff
@@ -134,7 +134,7 @@ under the License.
     <mockitoVersion>2.21.0</mockitoVersion>
     <plexusVersion>2.1.0</plexusVersion>
     <plexusInterpolationVersion>1.26</plexusInterpolationVersion>
-    <plexusUtilsVersion>3.4.2</plexusUtilsVersion>
+    <plexusUtilsVersion>3.3.1</plexusUtilsVersion>
```

Comment:
I would like to add a comments, issue number why this version is used, in order to intentionally  manage next updates.

Reply:
command added